hack/tools/internal/tilt-prepare/main.go

```diff
@@ -788,6 +788,7 @@ func writeIfChanged(prefix string, path string, yaml []byte) error {
 // This has the affect that the appended ones will take precedence, as those are read last.
 // Finally, we modify the deployment to enable prometheus metrics scraping.
 func prepareWorkload(name, prefix, binaryName, containerName string, objs []unstructured.Unstructured, ts *tiltSettings) error {
+	updateNamespacePodSecurityStandard(objs)
```

Comment:
```suggestion
	// Update provider namespaces to have the pod security standard enforce label set to privileged.
	// This is required because we remove the SecurityContext from provider deployments below to make tilt work.
	updateNamespacePodSecurityStandard(objs)

```